main.js

```diff
@@ -2,39 +2,44 @@ var $varImage = document.querySelector('img');
 var $carContainer = document.querySelector('.carStyle');
 
 
-var myCar ={
-  xCoordinate:'',
-  yCoordinate:'',
-  directions:''
+var myCar = {
+  xCoordinate: '',
+  yCoordinate: '',
+  directions: '',
+  motion: true
 };
 
-var intervalId=null;
-var distanceX=0;
-var distanceY=0;
+var intervalId = null;
+var distanceX = 0;
+var distanceY = 0;
 
-document.addEventListener('keydown',function(e){
-  clearInterval(intervalId);
-  $varImage.className ='';
-  if(e.keyCode ===39){
-    myCar.directions='east';
+document.addEventListener('keydown', function (e) {
+  $varImage.className = '';
+  if (e.keyCode === 39) {
+    myCar.directions = 'east';
     $varImage.classList.add(myCar.directions);
-  }else if(e.keyCode === 40){
-    myCar.directions ='south';
+  } else if (e.keyCode === 40) {
+    myCar.directions = 'south';
     $varImage.classList.add(myCar.directions);
-  }else if (e.keyCode ===37){
-    myCar.directions ='west';
+  } else if (e.keyCode === 37) {
+    myCar.directions = 'west';
     $varImage.classList.add(myCar.directions);
-  }else if (e.keyCode ===38){
-    myCar.directions= 'north';
+  } else if (e.keyCode === 38) {
+    myCar.directions = 'north';
     $varImage.classList.add(myCar.directions)
```

Comment:
How bout this
```
document.addEventListener('keydown', function (e) {
  $varImage.className = ''; // <- not sure what this is for
  if (e.keyCode === 39) {
    myCar.directions = 'east';
  } else if (e.keyCode === 40) {
    myCar.directions = 'south';
  } else if (e.keyCode === 37) {
    myCar.directions = 'west';
  } else if (e.keyCode === 38) {
    myCar.directions = 'north';
  }
    $varImage.classList.add(myCar.directions)

Reply:
got it! i set the className to nothing so that it cleared out the class when there was another keydown, but I changed the $varImage.classList.add ==> $varImage.className = myCar.directions and it worked! (don't need $varImage.className='' anymore).